manager/manager.go

```diff
@@ -112,12 +112,21 @@ func (rm *realManager) HousekeepModel() {
 }
 
 func (rm *realManager) Housekeep() {
+	for {
+		start := rm.lastSync
+		end := start.Add(rm.resolution)
+		timeToNextSync := end.Sub(time.Now())
+		// TODO: consider adding some delay here
```

Comment:
As for now scrapping data from sources takes place right after the end time passed. For example scrapping data for [10:01, 10:02] takes place a few milliseconds after 10:02. If the resolution is high some data may be missed so that we may want add some delay here in the future. I'm ok with removing this comment if you think it is useless.